<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>414</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>414</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="data_box">
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="data_layout"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="parameter_box">
     <property name="title">
      <string>Plot parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Color field</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsMapLayerComboBox" name="color_selection"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Color field type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="color_field_type">
        <item>
         <property name="text">
          <string>Continuous</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Categorical</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="n_categories_label">
        <property name="text">
         <string>N categories</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="n_categories"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_116">
        <property name="text">
         <string>Line type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="line_type">
        <item>
         <property name="text">
          <string>Curved</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Straight</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QgsColorButton" name="color">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>25</height>
         </size>
        </property>
        <property name="color">
         <color>
          <red>38</red>
          <green>162</green>
          <blue>105</blue>
         </color>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
